app/controllers/api/v1/rooms_controller.rb

```diff
@@ -35,7 +36,7 @@ def show
       private
 
       def find_room
-        @room = Room.find_by(friendly_id: params[:friendly_id])
+        @room = Room.find_by!(friendly_id: params[:friendly_id])
```

Comment:
If you're going to make this change, you should remove the `if else` from `rooms#show`

Reply:
Makes sense, I'll do it

Reply:
DONE,